packages/docs/src/routes/qwikcity/integrations/storybook/index.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: Qwik City and Storybook
+keywords: storybook, components, preview, testing, docs
+contributors:
+  - DustinJSilk
+---
+
+# Storybook
+
+Storybook is a frontend tool for building UI components and pages in isolation.
+It helps you develop hard-to-reach states and edge cases without needing to run
+the whole app. You can test and preview components in a local Storybook UI or
+deploy a preview of Storybook to a server to share with your team.
+
+## Usage
+
+You can add Storybook easily by using the following Qwik starter script:
+
+```shell
+npm run qwik add storybook
```

Comment:
Storybook usually gets added to applications with `npx storybook init`, which detects the framework it's being used with. Personally, I'd rather lean into supporting the existing tool rather than making a Qwik one that's different.
But if we can support both ways, that's okay too!